Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -173,12 +173,13 @@ extension Channel {
                 )
             }
         } else {
+            let unsafePosition = UnsafeTransfer(position)
```

Comment:
I don't think we should be doing this: `position` isn't sendable because it can contain a `ChannelHandler` which might not be `Sendable`. All we're doing here is suppressing a correct warning.

Reply:
Oh yeah this is not supposed to stay :) that's why this was just opened as draft. I just wanted to get the nightly pipeline to not fatal error (it's got warnings as errors enabled so it's been failing because of this for a while now) to double check the tests passed. But all of the pipelines seem broken right now anyways because of some Jenkins error I believe so I'll just revert this and trust in my local builds passing. 

We need to fix this somehow though, or remove the warns as errors flag temporarily.